metricbeat/include/list.go

```diff
@@ -25,6 +25,7 @@ import (
 	_ "github.com/elastic/beats/metricbeat/module/system/diskio"
 	_ "github.com/elastic/beats/metricbeat/module/system/filesystem"
 	_ "github.com/elastic/beats/metricbeat/module/system/fsstat"
+	_ "github.com/elastic/beats/metricbeat/module/system/load"
```

Comment:
@monicasarbu Probably the empty directories for load still exists from your other branch ;-)